eodag_labextension/config.py

```diff
@@ -7,11 +7,20 @@
 from __future__ import annotations
 
 from importlib.metadata import distributions
+from typing import Optional
 
 from pydantic import BaseModel, computed_field
 from pydantic_settings import BaseSettings, SettingsConfigDict
 
 
+class MapInfo(BaseModel):
+    """Map info model"""
+
+    tile_url: Optional[str] = None
+    tile_copyright: Optional[str] = None
```

Comment:
Those 2 settings should default to the OSM tile used by default with the labextension.